### DISCUSSION TITLE
Swapping hair, possible?

### DISCUSSION THREAD

**Discussion by daddypigg66** [glucauze/sd-webui-faceswaplab#102] [General]
Possibly the wrong people to ask (apologies), but can you swap an entire head, including the hair?
The face swapping is very good, but sometimes having the same/similar hair type helps.
Is there such a thing, or the only option is to train, etc (sorry probably a noob q).

Thx

**Comment by GunnyNinja** [comment]
Use inpainting with a description of the hair you want.

**Comment by glucauze** [comment]
No it's not possible, but you can try using controlnet reference on txt2img.
